### PR TITLE
Fix #14735 - Password | Input hover not displaying blocked cursor.

### LINKED ISSUE #14735: Input Components: Input hover not displaying blocked cursor when input is disabled.
### Describe the bug

When Password input is disabled it is not showing the "not-allowed" type cursor when hovering, instead it shows only the default cursor.

### Environment

Angular

### Reproducer

_No response_

### Angular version

18

### PrimeNG version

17.5.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

Latest

### Browser(s)

_No response_

### Steps to reproduce the behavior

1. Go to Disabled Password
2. Hover on the Disabled Password
3. It shows the default cursor.

### Expected behavior

1. Go to Disabled Password
2. Hover on the Disabled Password
3. It should be showing the block cursor.

### PR DESCRIPTION
**Pull request by VA-Ansari**
Fix #14735 : Password - Input hover not displaying blocked cursor.

### CODE REVIEW

File: src/app/components/common/common.css [hunk 1]
```diff
@@ -49,6 +49,10 @@
         pointer-events: none;
     }
 
+    .p-disabled-overlay{
+        cursor : not-allowed !important;
```
Comment (Vatspra) [inline src/app/components/common/common.css:53]: We should avoid using  !important. It would be better to use CSS Specificity .

Comment (cetincakiroglu) [reply src/app/components/common/common.css:53]: We cannot add `!important` to the component styles. Could you please remove and update the code?

**Comment by cetincakiroglu** [comment]
Hi,

Thanks for the effort and support!
Unfortunately, I have to close this PR because we need to update all input components and test them. This enhancement will be implemented by Core Team in v17.

**Comment by VA-Ansari** [comment]
You are welcome

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Pull request closed without merging**